### PR TITLE
[Release] Update index.json for extension [ spring ] [ spring-cloud ] 

### PR DESCRIPTION
**Pull request by azclibot**
[Release] Update index.json for extension [ spring ] [ spring-cloud ] 

 Triggered by Azure CLI Extensions Release Pipeline - ADO_BUILD_ID=1570091

Last commit against main: https://github.com/Azure/azure-cli-extensions/commit/43670c5cf6b6d21e87db6aa4f8b089b87b0231b8

### CODE REVIEW

**Comment by yonzhan** [comment]
spring-cloud

File: src/index.json [hunk 1]
```diff
@@ -27929,6 +27929,51 @@
...
+                    "generator": "bdist_wheel (0.30.0)",
+                    "license": "MIT",
+                    "metadata_version": "2.0",
+                    "name": "spring",
+                    "summary": "Microsoft Azure Command-Line Tools spring Extension",
+                    "version": "4.0.0"
+                },
+                "sha256Digest": "526c8e8993888da1591ce3561ddf91b5e4c0f030696e6671455a972a25515d9a"
+            }
+        ],
```
Comment (zhoxing-ms) [inline src/index.json:27976]:
```suggestion
```

Comment (zhoxing-ms) [reply src/index.json:27976]: Replaced by #4839

Comment (zhoxing-ms) [inline src/index.json:27976]:
```suggestion
```

Comment (zhoxing-ms) [inline src/index.json:27976]:
```suggestion
```

**Review by zhoxing-ms — APPROVED** [review]

File: src/index.json (hunk 1 above)
Comment (zhoxing-ms) [reply src/index.json]: Replaced by #4839

**Pull request merged**